### PR TITLE
Remove feature groups from RNA

### PR DESCRIPTION
**Pull request by david4096**
Modifies the current RNA schema to include filtering by feature ID and removes the `FeatureGroup` functionality. @diekhans @dcolligan @saupchurch @kozbo 

Also includes commits to keep RNA up to date against master.

### CODE REVIEW

File: src/main/proto/ga4gh/rna_quantification_service.proto [hunk 1]
```diff
@@ -180,37 +166,3 @@ message GetExpressionLevelRequest {
   string expression_level_id = 1;
 }
 
-// ****************  /featuregroups/search  *******************
```
Comment (david4096) [inline src/main/proto/ga4gh/rna_quantification_service.proto]: Note removal here.

File: src/main/proto/ga4gh/rna_quantification_service.proto [hunk 2]
```diff
@@ -142,11 +128,11 @@ message GetRnaQuantificationRequest {
 // This request maps to the body of 'POST /expressionlevels/search'
 // as JSON.
 message SearchExpressionLevelsRequest {
-  // If present return only ExpressionLevel records which belong to this group.
```
Comment (david4096) [inline src/main/proto/ga4gh/rna_quantification_service.proto]: Removing feature group ID from search request.

File: src/main/proto/ga4gh/rna_quantification.proto [hunk 3]
```diff
@@ -112,7 +97,4 @@ message ExpressionLevel {
   
   // Upper bound of the confidence interval on the expression value.
   float conf_interval_high = 11;
-
-  // Associated feature groups
-  repeated string feature_group_ids = 12;
```
Comment (david4096) [inline src/main/proto/ga4gh/rna_quantification.proto]: Remove feature_group_ids from expression level message.

**Comment by diekhans** [comment]
+1

**Comment by saupchurch** [comment]
+1

**Pull request merged**